index_fr.html

```diff
@@ -435,6 +426,69 @@ <h1 class="text-center mb-4 display-3">Livres</h1>
             <p><strong>A retirer uniquement à l'adresse 11-13 rue Pierre et Marie Curie, 75005 PARIS</strong></p>
 
         </div>
+        
```

Comment:
.ai file I uploaded by mistake, we do not need it. Can you delete *.ai file?

For the albums 1-6 I did no changes in the buttons, so they bring to paypal as previously.

For the new album 7 there is no need to go for paypal, because the album is electronic and for free. So I put the link to Amazon where people could download it. But otherwise I could maybe just wrote that books are free and available on Amazion. Let me know what you suggest is better to do.